sonar-helm-for-iac/src/main.go

```diff
@@ -63,6 +63,7 @@ func main() {
 
 func readAndValidateSources() (*converters.TemplateSources, error) {
 	scanner := bufio.NewScanner(os.Stdin)
+	scanner.Split(converters.ScanLinesIncludeNewLine)
```

Comment:
Maybe these two lines can be extracted into a function, not to repeat it in all of the tests.